lib/shortcodes/extra-faq.php

```diff
@@ -63,25 +63,29 @@ function extrafaq( $atts ) {
 		if ( $atts['subheadline'] ) {
 			?>
 			<div class="subhead--wrapper">
-				<p class="subhead"><?= esc_html( $atts['subheadline'] ); ?></p>
+				<p class="subhead"><?= $atts['subheadline'] // @codingStandardsIgnoreLine; ?></p>
```

Comment:
I think we should keep it escaped.

Reply:
NO there is a reason for it – we can use <br /> to split it to multiple lines which is not possible otherwise